fineract-provider/src/main/java/org/apache/fineract/portfolio/delinquency/service/DelinquencyWritePlatformServiceImpl.java

```diff
@@ -134,8 +146,72 @@ public CommandProcessingResult deleteDelinquencyBucket(Long delinquencyBucketId,
                         "Data integrity issue with resource: " + delinquencyBucket.getId());
             }
             repositoryBucket.delete(delinquencyBucket);
+            return new CommandProcessingResultBuilder().withCommandId(command.commandId()).withEntityId(delinquencyBucket.getId()).build();
```

Comment:
Unnecessary return. The delinquencyBucketId and the delinquencyBucket.getId() will be the same. no?

Reply:
Done, removed